### PR TITLE
Use sdk

### PR DESCRIPTION
**Pull request by smacker**
Lookout part of #218

Gopkg constraint uses revision instead of version. We will need to update it as soon as we release v0.0.4. (I believe we need to fix python first)

### CODE REVIEW

**Comment by smacker** [comment]
lookout failed. not sure if it's the right error because there is no PR id. But most probably it is:
```
{
"app":"lookoutd",
"error":"posting analysis failed: file not found: provider/github/poster.go",
"event-id":"73adba9b793078abb3e887bfc70d1dab4d4ea1dd",
"event-type":2,
"level":"error",
"msg":"event processing failed",
"source":"server/server.go:101",
"time":"2018-09-05T09:40:29.442824782Z"
}
```

File: .travis.yml [hunk 1]
```diff
@@ -10,13 +10,13 @@ git:
...
+      - ubuntu-toolchain-r-test
     packages:
-    - gcc-6
-    - g++-6
+      - gcc-6
+      - g++-6
 
 go:
-  - "1.10.x"
+  - '1.10.x'
```
Comment (bzz) [inline .travis.yml]: Why ' ?

**Comment by bzz** [comment]
Looking at vendor update made me wonder - is there a reason we do not prune non-go files?

File: .travis.yml (hunk 1 above)
Comment (smacker) [reply .travis.yml]: actually, my editor automatically reformatted the file and I didn't notice.
But in general, the difference between single quotes and double quotes is you don't need to escape `\`.
I can revert if you wish.

**Comment by smacker** [comment]
It wasn't pruned in prototype and we just continued using it like that. Maybe it's related to the need of proto files but even so we can enable pruning globally and disable for `google/protobuf` and `gogo` if we need. (but most probably we don't)

File: .travis.yml (hunk 1 above)
Comment (bzz) [reply .travis.yml]: 👍

**Comment by smacker** [comment]
Issue for prune: https://github.com/src-d/lookout/issues/222

**Pull request merged**